src/api/version.ts

```diff
@@ -224,7 +224,11 @@ export namespace VersionHelper {
       lastCheck: 0,
     }
     if (await fs.pathExists(newVersionInfoFilePath)) {
-      newVersionInfo = (await fs.readJson(newVersionInfoFilePath, { encoding: 'utf8' })) as NewVersionInfoData
+      try {
+        newVersionInfo = (await fs.readJson(newVersionInfoFilePath, { encoding: 'utf8' })) as NewVersionInfoData
+      } catch {
+        // file is corrupted
```

Comment:
could you output the content of the corrupted file ?
it may help to know what is corrupted

Reply:
I can put it in debug only.

Reply:
But anyway, I haven't found how to change outputlevel using env variables.
So, user won't see it anyway.